### PR TITLE
Cypress to ts

### PR DESCRIPTION
**Pull request by Sajma27**
# Summary
A description of the changes proposed in the pull request.

Fixes #number

## Checklist
- [ ] Change is small and easy to review (please split big changes into multiple PRs)
- [ ] Change is consistent with architecture
- [ ] Change is consistent with test architecture
- [ ] Change is consistent with naming conventions
- [ ] New code is covered with tests
- [ ] Tests related to old code are updated
- [ ] Documentation is up to date with changes

### CODE REVIEW

File: universal-login-wallet/cypress/integration/createAccount.test.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/// <reference path="../support/index.d.ts" />
```
Comment (xitronix) [inline universal-login-wallet/cypress/integration/createAccount.test.ts]: remove comment

File: universal-login-wallet/cypress.json [hunk 2]
```diff
@@ -1,6 +1,5 @@
 {
   "fixturesFolder": false,
-  "supportFile": false,
-  "pluginsFile": false,
+  "pluginsFile": "cypress/plugins/index.js",
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/cypress.json]: Do we need this line? I don't remember. Could you try to remove it?

File: universal-login-wallet/package.json [hunk 3]
```diff
@@ -32,6 +32,9 @@
     "react-transition-group": "^2.8.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.8.7",
+    "@babel/preset-env": "^7.8.7",
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/package.json]: We can also try to remove these, I don't think that we need `babel`

File: universal-login-wallet/package.json [hunk 4]
```diff
@@ -48,6 +51,7 @@
     "@typescript-eslint/eslint-plugin": "^2.0.0",
     "@typescript-eslint/parser": "^2.0.0",
     "@unilogin/relayer": "0.5.18",
+    "babel-loader": "^8.0.6",
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/package.json]: And this

File: universal-login-wallet/cypress/support/index.js [hunk 5]
```diff
@@ -0,0 +1,2 @@
+import './commands';
+import './LoginApp/commands';
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/cypress/support/index.js]: Why don't we name these commands just `loginPageCommands` and move it to `universal-login-wallet/cypress/support/loginPageCommands.js`? Then this file would be like this:
```js
import './commands';
import './loginPageCommands';
```

File: universal-login-wallet/cypress/integration/createAccount.test.ts (hunk 1 above)
Comment (Sajma27) [reply universal-login-wallet/cypress/integration/createAccount.test.ts]: This is not a comment, it won't work without it

File: universal-login-wallet/cypress/plugins/index.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
+/// <reference types="cypress" />
```
Comment (Marik-D) [inline universal-login-wallet/cypress/plugins/index.js]: Try to add `universal-login-wallet/cypress` directory to include section in local `tsconfig.json` that might remove the need to add these annotations

**Comment by JustynaBroniszewska** [comment]
Done in #2042

**Pull request closed without merging**